jax/_src/scipy/util.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import textwrap
+from typing import Callable, Optional, Sequence
+
+from jax._src.numpy.util import _wraps
+
+_no_chkfinite_doc = textwrap.dedent("""
+Does not currently support the Scipy argument ``check_finite=True``,
+because `jax.numpy.asarray_chkfinite` does not exist at the moment.
+""")
+
+_no_overwrite_doc = "Does not support the Scipy argument ``overwrite_*=True``."
+
+_no_overwrite_and_chkfinite_doc = _no_overwrite_doc + _no_chkfinite_doc
+
+def _no_overwrite_and_chkfinite_wraps(fun: Optional[Callable], update_doc: bool = True, lax_description: str = "",

```

Comment:
I don't think this utility is necessary - all it saves is passing a couple explicit parameters to `_wraps` at the call-site, yes? If so, I'd recommend deleting this an using `_wraps` directly.

Reply:
Okay, but the common docstring is useful right?

Reply:
Definitely, and you could put it in a global variable at the top of the file where it is used.